frontend/src/constants/labels.ts

```diff
@@ -1,30 +1,107 @@
-export const LABELS = {
-  patientInformation: {
-    title: "Patient Information",
-    items: [
-      { name: "First Name", required: true, color: '#FF0000' }, // red with 50% transparency
-      { name: "Last Name", required: true, color: '#00FF00' },  // green with 50% transparency
-      { name: "DOB", required: false, color: '#0000FF' },       // blue with 50% transparency
-      { name: "Gender", required: false, color: '#00FFFF' },    // cyan with 50% transparency
-      { name: "Phone Number", required: false, color: '#FF00FF' }, // magenta with 50% transparency
-      { name: "Email", required: false, color: '#FFFF00' },      // yellow with 50% transparency
-      {
-        name: "Address",
-        required: false,
-        color: '#EE82EE' // violet with 50% transparency
-      },
-    ],
-  },
-  organizationInformation: {
-    title: "Organization Information",
-    items: [
-      { name: "Name", required: true, color: '#00FF00' },        // lime with 50% transparency
-      { name: "Email", required: false, color: '#FF2400' },      // scarlet with 50% transparency
-      {
-        name: "Address",
-        required: false,
-        color: '#4B0082'  // indigo with 50% transparency
-      },
-    ],
-  },
+// Helper function to create label items
+const createLabelItem = (
+  id: string,
+  originalName: string,
+  isRequired: boolean,
+  color: string,
```

Comment:
Thought (not a change request) - it might be easier to have these colors assigned from a list/index in the `createLabelItem` function. 


So adding a `let labelIndex = 0` variable to the file and an array of colors. 

Then in here, if color is not passed in set color to `colorMap[labelIndex++]` to get an auto incrementing color map. 

You could also use a generator function if global variables give you bad feels 

```
const colorMapGenerator = function()* {
  let idx = 0
  while(true) {
    yield colorMap[idx++]
  }
} 
```

Then you can grab colors by instantiating an instance of the generator and calling `.next().value` on it for each item you want to have a color.